addons/website_slides_survey/models/slide_channel.py

```diff
@@ -27,3 +70,20 @@ def _remove_membership(self, partner_ids):
                 removed_channel_partner_domain)
             slide_partners_sudo.user_input_ids.slide_partner_id = False
         return super()._remove_membership(partner_ids)
+
+    @api.depends('channel_partner_ids')
+    def _compute_certified_members_count(self):
+        channels_count = self.env['slide.channel.partner'].sudo()._read_group(
+            domain=[('channel_id', 'in', self.ids),
+                    ('certification_passed', '=', True)],
+            groupby=['channel_id'],
+            aggregates=['partner_id:count_distinct']
+        )
+        mapped_data = {channel: count for channel, count in channels_count}
+        for channel in self:
+            channel.members_certified_count = mapped_data.get(channel, 0)
+
+    def action_redirect_to_certified_members(self):
+        action = self.action_redirect_to_members('certified')
+        action['help'] = "<p class='o_view_nocontent_smiling_face'>%s</p>" % _('No Attendee passed this course certification yet!')
```

Comment:
For a one-liner, this is probably more readable (and generally prefered).

```suggestion
        action['help'] = f"<p class='o_view_nocontent_smiling_face'>{_('No Attendee passed this course certification yet!')}</p>"
```

If you aim to improve readability however, I'd suggest

```suggestion
        help = _('No Attendee passed this course certification yet!')
        action['help'] = f"<p class='o_view_nocontent_smiling_face'>{help}</p>"
```

Reply:
I choose the most readable one :-)